specifyweb/frontend/js_src/lib/utils/cache/definitions.ts

```diff
@@ -89,11 +89,11 @@ export type CacheDefinitions = {
      * WorkBench column sort setting in a given dataset
      * {Collection ID}_{Dataset ID}
      */
-    [key in `${number}_${number}`]: RA<
-      hot.columnSorting.Config & {
-        readonly physicalCol: number;
-      }
-    >;
+    [key in `${number}_${number}`]: RA<{
+      readonly column: number;
+      readonly sortOrder: SortOrderType;
+      readonly physicalCol: number;
+    }>;
```

Comment:
Not sure if there is a better way to do this.  I tried using the newer path for the same type `Handsontable.plugins.ColumnSorting.Config` instead of `hot.columnSorting.Config` but that results in an error at line 235: 
```
/**
 * This will trigger a TypeScript type error if any cache definition
 * contains a value that is not JSON-Serializable.
 */
ensure<IR<IR<CacheValue>>>()({} as unknown as CacheDefinitions);
```

Reply:
Not sure why TypeScript complains, but this appears to satisfy it

```ts
    [key in `${number}_${number}`]: RA<Pick<hot.plugins.ColumnSorting.Config,'column'|'sortOrder'> & {
      readonly physicalCol: number;
    }>;
```

where `hot` is `import type hot from 'handsontable';`

and while I know that `Pick<hot.plugins.ColumnSorting.Config,'column'|'sortOrder'>` is quite verbose, in this case it's worth it as the presence of `column` and `sortOrder` (and their types) are determined by Handsontable, not us.
By using the type from them, our code would throw TypeScript error on Handsontable update when they change this type on their side in a way that is not compatible with our code - which is great as a TypeScript error is better than a run-time error caught by a user any time of the day